install/streams.csv

```diff
@@ -1035,7 +1035,9 @@ LTZ,20,1,false,false,false,2004-02-27T04:00:00Z,2013-10-23T05:00:00Z
 LTZ,20,50,false,false,false,2013-10-23T05:05:00Z,2019-07-01T23:15:00Z
 LTZ,20,200,false,false,true,2004-02-27T04:00:00Z,2019-07-01T23:15:00Z
 LTZ,20,200,false,false,false,2019-07-02T20:15:00Z,2019-10-31T21:44:00Z
-LTZ,21,200,false,false,false,2019-10-31T21:44:00Z,9999-01-01T00:00:00Z
+LTZ,21,50,false,false,false,2020-03-11T04:04:00Z,9999-01-01T00:00:00Z
+LTZ,21,200,false,false,false,2019-10-31T21:44:00Z,2020-03-11T02:21:00Z
+LTZ,21,200,false,false,false,2020-03-11T03:20:00Z,9999-01-01T00:00:00Z
```

Comment:
These two lines can be joined together as they don't add any new information.

Reply:
Thanks for the review. Entry for 200hz remains unchanged now.